spec/integration/commands/service_command/create_command_spec.rb

```diff
@@ -0,0 +1,67 @@
+require '3scale_toolbox'
+
+RSpec.describe 'Service Create command' do
+  include_context :real_api3scale_client
+  include_context :random_name
+  subject { ThreeScaleToolbox::CLI.run(command_line_str.split) }
+  let(:remote) do
+    endpoint_uri = URI(endpoint)
+    endpoint_uri.user = provider_key
+    endpoint_uri.to_s
+  end
+
+  context 'with the specified service not existing' do
+    let (:service_name) { "service_#{random_lowercase_name}" }
+    let (:service_ref) { service_name }
+
+    context 'without specifying options' do
+      let (:command_line_str) { "service create #{remote} #{service_name}" }
+
+      it "successfully creates a new service" do
```

Comment:
I do not know if it makes sense. But, what about checking service options are expected default ones? WDYT?